app/_data/docs_nav_kuma_2.1.x.yml

```
@@ -0,0 +1,897 @@
+product: kuma
+release: 2.1.x
+items:
```

```suggestion
generate: true
items:
```
@Automaat I think we're missing this which is why it won't generate the nav